test/aqua/test_optimizer_aqqd.py

```diff
@@ -44,8 +44,8 @@ def test_aqgd(self):
 
         result = VQE(self.qubit_op,
```

Comment:
I think there should be additional testing around the new functionality to assure it works - such as supplying a list of values in maxiter, tests that checks that error paths behave as expected etc.

Does this updated version perform differently than the prior version? I agree that the test need not be so precise but was this a needed change to the number of places?

Reply:
Added more tests. Since the convergence test logic has been changed we can not assume the same results precisely. That's why it was changed.